specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1564,922 +2801,1226 @@
-          "description": "Indicates the time of the day. True indicates 'day',', false indicates 'night."
-        },
-        "cloudCover": {
-          "type": "integer",
-          "format": "int32",
-          "description": "Percent representing cloud cover."
-        },
-        "temperature": {
-          "$ref": "#/definitions/WeatherUnit"
+          "description": "Datetime the forecast is valid, displayed in ISO8601 format."
         },
-        "wind": {
-          "$ref": "#/definitions/WindDetails"
+        "initializedDateTime": {
+          "type": "string",
+          "description": "Datetime the forecast was created, displayed in ISO8601 format."
         },
-        "windGust": {
-          "$ref": "#/definitions/WindDetails"
+        "location": {
+          "$ref": "../../../Common/preview/1.0/common.json#/definitions/LatLongPair",
+          "description": "Coordinates of the storm"
         },
-        "precipitation": {
-          "$ref": "#/definitions/WeatherAlongRoutePrecipitation"
+        "maxWindGust": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Maximum wind gust speed associated with the storm. May be NULL."
         },
-        "lightningCount": {
-          "type": "integer",
-          "format": "int32",
-          "description": "Estimation of thunderstorm intensity on an open scale. A value of 0 means there is no thunderstorm; values of 1 and higher mean there is a thunderstorm in increasing intensity."
+        "sustainedWind": {
+          "$ref": "#/definitions/WeatherUnit",
+          "description": "Maximum sustained wind speed associated with the storm. May be NULL."
         },
-        "sunGlare": {
-          "$ref": "#/definitions/SunGlare",
-          "description": "A rating that indicates how blinding the sun is for the driver."
+        "status": {
+          "type": "string",
+          "description": "Storm status"
         },
-        "hazards": {
-          "$ref": "#/definitions/WeatherHazards"
+        "window": {
+          "$ref": "#/definitions/Window",
+          "description": "Contains information about the forecast window for the storm during the specified time period (not the entire cone). If windowGeometry=true in the request, this object will include geoJSON details for window geometry."
         },
-        "notifications": {
-          "description": "List of weather hazard notifications.",
+        "windRadiiSummary": {
           "type": "array",
           "items": {
-            "$ref": "#/definitions/WeatherNotification"
-          }
+            "$ref": "#/definitions/StormWindRadiiSummary"
+          },
+          "description": "Displayed when details=true or radiiGeometry=true in the request."
         }
       }
     },
-    "WeatherHazards": {
-      "description": "Description of the weather hazard affecting the trip.",
+    "Window": {
       "type": "object",
+      "description": "Government-issued storm forecast",
```

Comment:
Change:
Government-issued storm forecast

To:
Contains information about the forecast window for the storm during the specified time period (not the entire cone). If windowGeometry=true in the request, this object will include geoJSON details for window geometry.